### ISSUE TITLE
Need to improve help file

### ISSUE THREAD

**Issue by RB61** [lavsen25/MIRA-MISA-Lab-3-#1]
Thanks for the code. But, it is not clear where to start.  It seems that main.m just displays final results. 
I am getting this error:
The file resultsNIIAtlas/resultsafterTransformixCSF/1004/result.nii does not exist

Thanks for helping.

**Comment by lavsendahal** [comment]
Yes you need to have the atlas of all brain tissues at the path provided in the code. I don't have rights to upload data.
